packages/blade/src/components/Tooltip/__tests__/Tooltip.web.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('<Tooltip />', () => {
 
   it('should render tooltip with custom zIndex', () => {
     const buttonText = 'Hover me';
-    const { container, getByRole, queryByRole } = renderWithTheme(
+    const { baseElement, getByRole, queryByRole } = renderWithTheme(
```

Comment:
expected change?

Reply:
Yes, we want to snapshot the open tooltip and not the trigger.